File: Day-1/program4.c
#include<stdio.h>
int main()
{//opening main
    int c=0,n;//initialising variable
    printf("enter number");
    scanf("%d",&n);//storee input in variable
    if(n==0)
{
    c=1;
}
    while (n!=0)
    {//opening while loop
     n=n/10;
     c++;
    }//closing while loop
  printf("%d",c);
  return 0;  
}//closing main

File: Day-11/program4.java
import java.util.Scanner;
public class program4
{//opening class
    static int fact(int n) //FUNCTION TO FIND FACTORIAL
{
    int p=1;
        for(int i=1;i<=n;i++)
        {
            p=p*i;
        }
        return p;
    }
    public static void main(String[] args) 
    {//opening main
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter number: ");
        int n= sc.nextInt();//initialising
        System.out.println("FACTORIAL OF INPUT NUMBER="+fact(n));//calling function
    sc.close();
}//closing main
}//closing class

